os/board/rtl8720e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -186,8 +186,8 @@ static void cmd_wifi_sta_and_ap(int argc, char **argv)
 {

```

Comment:
Could you please add all the WIFI related files under CONFIG_AMEBALITE_WIFI flag.
So that in case we need to test the code after disabling CONFIG_AMEBALITE_WIFI & wifi manager, build does not break

Reply:
If you could verify it at your end as well. There is a lot of dependency with CONFIG_AMEBALITE_WIFI flag like:
rtk_netmgr.c
etc

Reply:
Dear @thapav , sure can provide, but as what I can see currently it has many compiling errors after disabling wifi & wifi manager flags in defconfig. I will fix them and push new code here.

Reply:
Dear @thapav ，new codes have been pushed. Now if you disable CONFIG_AMEBALITE_WIFI and all wifi manager flags(except CONFIG_WIFIMGR_STA_IFNAME="wlan0") in defconfig file, build should be ok.